packages/nextjs/hooks/useCohortAddBuilderEvents.ts

```diff
@@ -0,0 +1,25 @@
+import { gql, useQuery } from "urql";
+import contracts from "~~/generated/hardhat_contracts";
+
+const BuildersQuery = gql`
+  query Builders($cohortAddress: String!) {
+    cohortBuilders(where: { cohortContractAddress: $cohortAddress }, orderBy: "timestamp", orderDirection: "desc") {
+      amount
+      ens
+      id
+      timestamp
```

Comment:
I think we only need the id here right?

Reply:
Ups, added all fields in autopilot mode! Fixed